src/controllers/livetvtuner.html

```diff
@@ -38,6 +38,11 @@ <h1 class="sectionTitle">${HeaderLiveTvTunerSetup}</h1>
                     <div class="fieldDescription">${SimultaneousConnectionLimitHelp}</div>
                 </div>
 
+                <div class="inputContainer fldFallbackMaxStreamingBitrate hide">
+                    <input is="emby-input" type="number" pattern="[0-9]*" required="required" min="0" step="1" class="txtFallbackMaxStreamingBitrate" label="${LabelFallbackMaxStreamingBitrate}" autocomplete="off" value="30" />
```

Comment:
Currently, we can pass `0` and it is saved correctly. But it's actually an invalid value - [Min(..., 0)](https://github.com/jellyfin/jellyfin/blob/62712aa12cdc88c524923cb318783b23a8a7a2c5/Jellyfin.Api/Helpers/StreamingHelpers.cs#L152). Moreover, we [use `30` in case of `0`](https://github.com/jellyfin/jellyfin-web/blob/10a8dead0caae9b99f8a08e9b2862b9861dfc29d/src/controllers/livetvtuner.js#L67) in web.

Maybe use `1` as a minimum?
```suggestion
                    <input is="emby-input" type="number" pattern="[0-9]*" required="required" min="1" step="1" class="txtFallbackMaxStreamingBitrate" label="${LabelFallbackMaxStreamingBitrate}" autocomplete="off" value="30" />
```

Reply:
I think it might be more reasonable to do such thing on the server side because not only this field might pass incorrect value, and any client/plugins can also set some unreasonable values. But it does not hurt to set the lower limit to 1 on web as well.